modules/fbx/fbx_document.cpp

```diff
@@ -1266,7 +1266,7 @@ Error FBXDocument::_parse_cameras(Ref<FBXState> p_state) {
 			camera->set_fov(Math::deg_to_rad(real_t(fbx_camera->field_of_view_deg.y)));
 		} else {
 			camera->set_perspective(false);
-			camera->set_size_mag(real_t(fbx_camera->orthographic_size.y));
+			camera->set_size_mag(real_t(fbx_camera->orthographic_size.y * 0.5f));
```

Comment:
This change is fully orthogonal to this PR, I just included it as it fixes the test case here.

Let me know if you would rather have such things in a separate PR.

Reply:
I think it's okay to include multiple bugfixes here. It's clear they are fixing different issues because they are separate commits.